docs/react/guides/storybook-plugin.md

```diff
@@ -132,11 +136,15 @@ To register an [addon](https://storybook.js.org/addons/) for all storybook insta
 
 To register an [addon](https://storybook.js.org/addons/) for a single storybook instance, go to that project's `.storybook` folder:
 
-1. In `addons.js` add the register import statement.
+1. In `main.js`, in the `addons` array of the `module.exports` object, add the new addon:
    ```
-   import '@storybook/addon-knobs/register';
+   module.exports = {
+   stories: [...],
+   ...,
+   addons: [..., '@storybook/addon-knobs/register'],
```

Comment:
I think this might need more consideration...

In SB6 they are encouraging people to move away from knobs, towards the (now built in and) more powerful "controls" addon.

It works really nicely with the docs addon and gives you interactive controls which make sense for the input type (e.g. dropdown for an enum). 

You can see "controls" and docs in action on SB's own design system here: https://5ccbc373887ca40020446347-vehbzpmjqt.chromatic.com/?path=/docs/design-system-button--all

More information about the move away from knobs towards controls: https://github.com/storybookjs/storybook/tree/next/addons/controls

---

As an aside, even if knobs ends up staying here, it is now idiomatic to not have `/register` at the end of the path, because SB handles calling the register hook automatically as of v6 I believe

Reply:
Please note, you have instances of the /register throughout this PR, I won't tag them all individually

Reply:
UPDATE: I found the docs on the register point

The register is only needed if you want to manually configure the decorator. If you want to do the standard preset behaviour, you omit it.

Info I found is here: https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#new-addon-presets

Reply:
> I think this might need more consideration...
> 
> In SB6 they are encouraging people to move away from knobs, towards the (now built in and) more powerful "controls" addon.
> 
> It works really nicely with the docs addon and gives you interactive controls which make sense for the input type (e.g. dropdown for an enum). 
> 
> Unlike knobs, you also do not need to manually instrument the controls with utility functions in most cases. They "just work".
> 
> You can see "controls" and docs in action on SB's own design system here: https://5ccbc373887ca40020446347-vehbzpmjqt.chromatic.com/?path=/docs/design-system-button--all
> 
> More information about the move away from knobs towards controls: https://github.com/storybookjs/storybook/tree/next/addons/controls
> 

Hey @JamesHenry 👋

Yeah me and Katerina were briefly discussing the move to controls / args last week since they seem to be the way to go from now onwards.

Reply:
Hey @JamesHenry ! It's great that you have so much insight, let's find a time when you can help us out on this!